src/app/pages/TransactionDetailPage/index.tsx

```diff
@@ -82,12 +83,17 @@ export const TransactionDetailPage: FC = () => {
         </SubPageCard>
       )}
       {transaction && (
-        <SubPageCard title={t('transaction.header')} key={`${transaction.round}_${transaction.index}`}>
-          {!isMobile && <Divider variant="fullWidth" />}
-          <StyledDescriptionList titleWidth="200px">
+        <SubPageCard
+          featured
+          title={t('transaction.header')}
+          key={`${transaction.round}_${transaction.index}`}
+        >
+          <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
             <dt>{t('common.hash')}</dt>
             <dd>
-              {isMobile ? trimLongString(transaction.hash) : transaction.hash}
+              <Typography variant="mono" component="span" sx={{ color: COLORS.brandDark, fontWeight: 700 }}>
```

Comment:
Should these be links to itself, instead of just looking like links? Or should it copy the text? @donouwens

Reply:
So currently, 
 - the hash is formatted as a link ✔️ 
 - the string itself is not a link, so clicking on it does nothing
 - besides the string, we have a "copy to clipboard" button, which does copy it to the clipboard. ✔️ 

Is that OK, @donouwens ?

Reply:
OK, so we can close this.